src/pymordemos/output_error_estimation.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
This demo feels more like a test script to me. I don't thinks it is useful for the user in any way. In view of later adding a DWR output error estimate, I think it makes sense to keep this demo to compare the estimates. However, instead of dumping a few numbers, we should have some plot which compares the estimators. The FOM should be selectable by the user and so on.

Reply:
Yes it is actually true that this works more like a test. Accidentally ended to be a demo, probably ;). A plot and a more user friendly script is definitely something that we should do.